tests/test_indexer.py

```diff
@@ -461,6 +497,13 @@ def test_case(verify_payloads, **endpoint):
 
     @testmode.standalone
     def test_subscription_with_attachments(self):
+        remaining_time = SpecificRemainingTime(300)
+        backend = CompositeIndexBackend(executor=self.executor,
+                                        backends=DEFAULT_BACKENDS,
+                                        remaining_time=remaining_time,
+                                        notify_async=testmode.is_integration(),
+                                        notify=True)
```

Comment:
Can you extract this into a method? That would also make it more obvious what the difference is in the backend confiugration between the individual tests.